protocol/app/app.go

```diff
@@ -994,9 +994,8 @@ func New(
 	clobFlags := clobflags.GetClobFlagValuesFromOptions(appOpts)
 	logger.Info("Parsed CLOB flags", "Flags", clobFlags)
 
-	memClob := clobmodulememclob.NewMemClobPriceTimePriority(
-		app.IndexerEventManager.Enabled() || app.GrpcStreamingManager.Enabled(),
-	)
+	memClob := clobmodulememclob.NewMemClobPriceTimePriority(app.IndexerEventManager.Enabled())
+	memClob.SetGenerateOrderbookUpdates(app.GrpcStreamingManager.Enabled())
```

Comment:
we should probably also get rid to the `app.IndexerEventManager.Enabled()` in the builder method and use the `Set...` method instead

Reply:
hard to add arguments to the constructor method since we have a ton of tests that need to be modified